packages/api/helper/mailer.js

```diff
@@ -33,7 +34,7 @@ const email = new emailTemplate({
   transport: transporter,
   send: true,
   //send status will eventually need to be updated to true
-  preview: true,
+  preview: false,
```

Comment:
What's the reason for this change?

Reply:
Its related to the email previewer. I believe it was initially meant to be false, and for testing purposes anyone can turn it back on.